charts/images.yaml

```diff
@@ -220,4 +220,4 @@ images:
 - name: hvpa-controller
   sourceRepository: github.com/gardener/hvpa-controller
   repository: eu.gcr.io/gardener-project/gardener/hvpa-controller
-  tag: "0.1.0"
+  tag: "v0.2.0
```

Comment:
I guess with this upgrade all other Hvpa manifests should also be adapted, right?
Can you please adapt `charts/gardener/controlplane/charts/runtime/templates/apiserver/hvpa.yaml` accordingly?
Please also decrease the limits as discussed [here](https://github.com/gardener/gardener/pull/1880#discussion_r374526808)